include/secp256k1_extrakeys.h

```diff
@@ -7,6 +7,75 @@
 extern "C" {
 #endif
 
+/** Opaque data structure that holds a parsed and valid "x-only" public key.
+ *  An x-only pubkey encodes a point whose Y coordinate is even. It is
+ *  serialized using only its X coordinate (32 bytes). See BIP-340 for more
+ *  information about x-only pubkeys.
```

Comment:
I think this source code is not consistent with the BIP340 document. The BIP talks about going with the option of implicitly choosing the Y coordinate that is a quadratic residue, not choosing the even Y coordinate.